GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -24,6 +24,7 @@ public sealed partial class FormFileHistory : GitModuleForm
         private const string FormBrowseName = "FormBrowse";
 
         private readonly TranslationString _buildReportTabCaption = new("Build Report");
+        private readonly TranslationString _noChanges = new("No changes.");
```

Comment:
`FileStatusList` has this without the trailing dot.
```suggestion
        private readonly TranslationString _noChanges = new("No changes");
```

Reply:
Will use a common string (without a trailing dot)